integration-tests/test/test_unbond.py

```diff
@@ -81,8 +111,11 @@ def test_unbond_validator(command_line_options: CommandLineOptions, random_gener
 
         # block number 4
         # withdraw not happen yet
-        validator_1_balance = get_vault_balance(context, bootstrap_node, VALIDATOR_KEY_1.get_public_key().get_rev_address(), VALIDATOR_KEY_2, 100000, 1)
-        assert validator_1_balance < 20000000
+        validator_1_balance_before_refund = get_vault_balance(context, bootstrap_node,
+                                                              VALIDATOR_KEY_1.get_public_key().get_rev_address(),
+                                                              VALIDATOR_KEY_2, 100000, 1)
+        # the unbond process cost validator1 some phlos so the balance of validator1 is smaller than the initial wallet amount
+        assert validator_1_balance_before_refund < validator_1_initial_wallet_amount
```

Comment:
Is there no way to get the precharge amount (say `validator_1_initial_precharge_amount`) and assert the exact difference, i.e.
`assert validator_1_balance_before_refund == validator_1_initial_wallet_amount - validator_1_initial_precharge_amount`?